platooning_strategic_IHP/include/platoon_strategic_ihp.h

```diff
@@ -267,7 +304,7 @@ namespace platoon_strategic_ihp
             std::shared_ptr<carma_wm::WMListener> wml_;
             carma_wm::WorldModelConstPtr wm_;
 
-            // local copy of configuration file
+            // local copy of pose
```

Comment:
I don't understand this change.  It seems the old comment was quite accurate.

Reply:
bad version control, updated comment.